thanos/templates/query-deployment.yaml

```diff
@@ -39,18 +39,25 @@ spec:
         args:
         - "query"
         - "--log.level={{ .Values.query.logLevel }}"
-        - "--grpc-address=0.0.0.0:{{ .Values.query.grpc.port }}"
+        {{- if .Values.cluster.enabled }}
         - "--cluster.address={{ .Values.cluster.address }}:{{ .Values.cluster.port}}"
         - "--cluster.peers={{ include "thanos.fullname" . }}.{{ .Release.Namespace }}.svc.cluster.local:{{ .Values.cluster.port }}"
+        {{- end }}
+        - "--grpc-address=0.0.0.0:{{ .Values.query.grpc.port }}"
         - "--query.replica-label={{ .Values.replicaLabelName }}"
         - "--http-address=0.0.0.0:{{ .Values.query.http.port }}"
+        {{- if .Values.query.extraArgs }}
+        {{ toYaml .Values.query.extraArgs | nindent 8 }}
+        {{- end }}
```

Comment:
Please remove these extraArgs because of #908

Reply:
Thanks, but I had a confusion with the #908 PR. I removed these changes from it - it should be merged here, as this is how it was tested.